raspibolt-pulse-switch.sh

```diff
@@ -125,6 +129,15 @@ fi
 network_rx=$(ifconfig ${network_name} | grep 'RX packets' | awk '{ print $6$7 }' | sed 's/[()]//g')
 network_tx=$(ifconfig ${network_name} | grep 'TX packets' | awk '{ print $6$7 }' | sed 's/[()]//g')
 
```

Comment:
I'm running Debian 11 which does ship ifconfig by default. Using ip instead.

#### Debian 
#network_rx=$(ip -s -h link show ${network_name} | grep RX: -A1 | tail -n 1 | awk '{ print $1 }')
#network_tx=$(ip -s -h link show ${network_name} | grep TX: -A1 | tail -n 1 | awk '{ print $1 }')

Reply:
Yeah different OS / machines could also require different network interface or temperature sensor.

Reply:
Regarding `ifconfig`, we could include in the list of programs to install which is at the start of the [bonus guide](https://raspibolt.org/guide/bonus/raspberry-pi/system-overview.html). For now we install `jq`, `netcat `and `net-tools`, we could add `ifconfig`:
![image](https://user-images.githubusercontent.com/86242283/191525588-0165a9f5-2f8b-4a7a-8546-c1a51a418446.png)

Wdyt?

Reply:
these lines work without additional installation:
```
network_rx=$(ifconfig ${network_name} | grep 'RX packets' | awk '{ print $6$7 }' | sed 's/[()]//g')
network_tx=$(ifconfig ${network_name} | grep 'TX packets' | awk '{ print $6$7 }' | sed 's/[()]//g')
```
so i would stick to them. If someone wants to install special packages to execute similar commands we could give a hint but should not make it mandatory to install more packages than necessary imo.

Reply:
I just read this: https://linuxconfig.org/how-to-install-missing-ifconfig-command-on-debian-linux

They state:

>  The ifconfig is deprecated. We highly recommend that you eventually grow accustomed to its replacement, the ip command.

I propose we update the script and replace the deprecated `ifconfig` by `ip` instead. Wdyt? We don't have to do this rn, but I could create an Issue as a reminder to do this in the near future.

Reply:
Ok, if its deprecated we should switch to `ip`